core/actions/slackPreviewRelease.js

```diff
@@ -28,20 +27,6 @@ module.exports = function createPreviewRelease(getConfig, getBranchStatus, getRe
     }, cb);
   }
 
-  function getBranchStatusForEachRepo(reposInfo, cb) {
-    logger.debug('getBranchStatusForEachRepo', { reposInfo });
-    mapSeries(reposInfo, (repoInfo, nextRepo) => {
-      const devBranch = get(repoInfo, 'config.github.devBranch');
-      getBranchStatus(repoInfo.repo, devBranch, (err, sha) => {
-        if (err) {
-          logger.error(`Error getting repo branch status for ${repoInfo.repo} ${devBranch}`, repoInfo.repo, devBranch, err);
-          return nextRepo(null, { repo: repoInfo.repo, failReason: 'INVALID_REPO_STATUS' });
-        }
-        nextRepo(null, Object.assign({}, repoInfo, { sha }));
```

Comment:
we can't remove the getBranchStatus step because it gets the sha that is used by the code build job